templates/logosSection.hbs

```diff
@@ -0,0 +1,16 @@
+<section class="aliados-wrapper">
+  <div class="aliados-section">
+    <div class="aliados-section__eyebrow">
+      <p class='eyebrow-typography aliados-section__eyebrow--title'>{{eyebrow}}</p>
+      <h3 class='heading-3 aliados-section__eyebrow--description'> {{title}}</h3>
+    </div>
+    <div class='aliados-section__logos'>
+      {{#each logos}}
+        <img class="aliados-section__logos--img" src="{{assetSrc this}}" alt="{{assetTitle this}}" />
+      {{/each}}
+    </div>
+  </div>
+
+  <img class='aliados-wrapper__shape-a' src="/image/Shape-A.svg" alt="shape-A">

```

Comment:
In this case I think you can use alt="" this images are decorative and don't need read when using a screen reader.<